frontend/src/components/navbar.svelte

```diff
@@ -80,12 +80,12 @@
             on:click={() => ($chosenTheme = theme.value)}
           >
             {#if $chosenTheme == theme.value}
-              <a class="bg-primary hover:bg-primary">
+              <a href="/" class="bg-primary hover:bg-primary">
```

Comment:
the href is back :O

Reply:
delete it, but now we have the warnings back

Reply:
Let's handle the warning in a different issue :DD